.github/workflows/continuous-integration.yml

```diff
@@ -22,12 +22,8 @@ jobs:
       - name: detect changed files
         run: |
           INTERESTING_FILES=$(
-            sort -u < <(
-              # Changes that are not committed (staged + unstaged + untracked), but without deleted files
-              git status --short -v -v --no-renames --porcelain | awk '$1 != "D" { print $2 }'
-              # Changes committed so far (may overlap with the above)
-              git diff --no-renames --name-only --diff-filter=d ${{ github.sha }}^
-            )
+            # Changes committed to this PR so far
+            git diff --no-renames --name-only --diff-filter=d ${{ github.sha }}^
           )
```

Comment:
no need to have new lines at all:

```sh
# Changes committed to this PR so far
INTERESTING_FILES=$( git diff --no-renames --name-only --diff-filter=d ${{ github.sha }}^ )
```

(can't post a direct suggestion since it covers both original and changed lines)